src/chttpd_db.erl

```diff
@@ -449,6 +449,45 @@ db_req(#httpd{method='POST',path_parts=[_,<<"_bulk_docs">>], user_ctx=Ctx}=Req,
 db_req(#httpd{path_parts=[_,<<"_bulk_docs">>]}=Req, _Db) ->
     send_method_not_allowed(Req, "POST");
 
+
+db_req(#httpd{method='POST',
+              path_parts=[_, <<"_bulk_get">>],
+              mochi_req=MochiReq}=Req,
+       Db) ->
+    couch_stats:increment_counter([couchdb, httpd, bulk_requests]),
+    couch_httpd:validate_ctype(Req, "application/json"),
+    {JsonProps} = chttpd:json_body_obj(Req),
+    case couch_util:get_value(<<"docs">>, JsonProps) of
+    undefined ->
+        throw({bad_request, <<"Missing JSON list of 'docs'.">>});
+    Docs ->
+        #doc_query_args{
+            options = Options
+        } = bulk_get_parse_doc_query(Req),
+
+        AcceptMixedMp = MochiReq:accepts_content_type("multipart/mixed"),
+        AcceptRelatedMp = MochiReq:accepts_content_type("multipart/related"),
+
+        case AcceptMixedMp orelse AcceptRelatedMp of
+        true ->
+            throw({not_acceptable,
```

Comment:
Let's remove it, it's the wrong http semantic and it's best not to have speculative / non-working code (we have enough of that by accident!)